plugins/modules/sonic_system.py

```diff
@@ -83,6 +83,20 @@
         choices:
           - ENABLE
           - DISABLE
+      load_share_hash_algo:
+        description:
+          - Specifies different types of ECMP Load-share hash algorithm
```

Comment:
Please delete the hyphen in "Load-share". I think it may be causing YAML parsing problems. You could enclose the two words in quotes as shown below:

```suggestion
          - Specifies different types of ECMP "Load share" hash algorithm
```

but the safest option would be to do it like this:

```suggestion
          - Specifies different types of ECMP Load share hash algorithm
```

Reply:
Ok. I committed the change